common/persistence/nosql/nosqlplugin/cassandra/gocql/client.go

```diff
@@ -154,14 +158,45 @@ func NewCassandraCluster(
 	return cluster, nil
 }
 
-// regionHostFilter returns a gocql host filter for the given region name
-func regionHostFilter(region string) gocql.HostFilter {
+func createHostFilter(
+	datacenter string,
+	enableContactPointHostFilter bool,
+	hosts []string) gocql.HostFilter {
+	var datacenterHostFilter gocql.HostFilter
+	var contactPointHostFilter gocql.HostFilter
+
+	if enableContactPointHostFilter {
+		allHosts := make([]string, 0)
+		for _, host := range hosts {
+			allHosts = append(allHosts, host)
+			addrs, err := net.LookupHost(host)
```

Comment:
it looks like we're not logging an error here if it occurs. Is this intentional?